plugins/module_utils/network/sonic/facts/system/system.py

```diff
@@ -86,6 +86,22 @@ def get_anycast_addr(self):
             data = {}
         return data
 
+    def get_auto_breakout(self):
+        """Get auto-breakout status available in chassis"""
+        request = [{"path": "data/sonic-device-metadata:sonic-device-metadata/DEVICE_METADATA/DEVICE_METADATA_LIST=localhost", "method": GET}]
+        try:
+            response = edit_config(self._module, to_request(self._module, request))
+        except ConnectionError as exc:
+            self._module.fail_json(msg=str(exc), code=exc.code)
+        if ('sonic-device-metadata:DEVICE_METADATA_LIST' in response[0][1]):
+            auto_breakout_data = response[0][1]['sonic-device-metadata:DEVICE_METADATA_LIST']
+            if 'auto-breakout' in auto_breakout_data[0]:
+                auto_breakout_val = auto_breakout_data[0]['auto-breakout']
+                data = {'auto-breakout': auto_breakout_val}
+            else:
+                data = {}
```

Comment:
If line#96 is false through, data will not have definition. So data can be initiated before line#96 and remove line#101 and 102.

Reply:
> If line#96 is false-through, the "data" will not have definition. The "data" can be initiated before line#96 and remove line#101 and 102.

Agreed. Updated the code.